holmes/plugins/toolsets/runbook/runbook_fetcher.py

```
@@ -11,38 +12,95 @@
+    get_runbook_by_path,
+    load_runbook_catalog,
+    DEFAULT_RUNBOOK_SEARCH_PATH,
+)
 from holmes.plugins.toolsets.utils import toolset_name_for_one_liner
 
 
 # TODO(mainred): currently we support fetch runbooks hosted internally, in the future we may want to support fetching
 # runbooks from external sources as well.
 class RunbookFetcher(Tool):
     toolset: "RunbookToolset"
+    available_runbooks: List[str] = []
```

Ruff (0.13.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)